docs/ingress-resources.md

```diff
@@ -110,6 +109,8 @@ Optional annotations are:
 
 - **ssl-policy**: Defines the [Security Policy](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/create-https-listener.html#describe-ssl-policies) that should be assigned to the ALB, allowing you to control the protocol and ciphers.
 
+- **alb.ingress.kubernetes.io/actions.\<ACTION NAME>**: Provides a method for configuring custom actions on a listener, such as for [Redirect Actions](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-listeners.html#redirect-actions). The `<ACTION NAME>` in the annotation must match the `serviceName` in the ingress rules. The value of the annotation is the JSON spec of the action. See the [Action type](https://docs.aws.amazon.com/sdk-for-go/api/service/elbv2/#Action) for documentation on what should be in the JSON. An example for a fixed-response would be: `alb.ingress.kubernetes.io/actions.fixed-response-error: '{"Type": "fixed-response", "FixedResponseConfig": {"ContentType":"text/plain", "StatusCode":"503", "MessageBody":"503 error text"}}'` for a `serviceName` of `fixed-response-error`. _NOTE_ you must set the `servicePort` to `use-annotation`.
```

Comment:
I think it's better to have examples and notes of `fixed-respone` and `redirect` clearly in sub-indented bullet points.

Reply:
One of the main reasons we're waiting for this is http -> https redirection. Which I'd say is the biggest use case so far. Maybe that should be a redirect example?